### ISSUE TITLE
3.8.1 Regression Issue - MediaLibrary is no longer available on iOS

### ISSUE THREAD

**Issue by SimonDarksideJ** [MonoGame/MonoGame#7940]
In migrating the MonoGame.Samples iOS project, an issue was hit with the use of MediaPlayer.

On all other platforms, MediaPlayer is used to play background audio and mp3 files, however, in the upgrade to 3.8.1, MediaPlayer is no longer loadable.

Any use of Media player is met with references not found (although the namespace is still available) or static properties / functions not being available, such as:

```
	MediaPlayer.IsRepeating = true;
	MediaPlayer.Play(Sound.Music);
```

In short, MediaPlayer is currently not usable for iOS.

To replicate, simply:
* Create a new MonoGame.iOS project
* Add the Media Namespace (Microsoft.Xna.Framework.Media) to Game1.cs
* Add the above code in a method in Game1.cs
* Observe the red squiggles

#### What version of MonoGame does the bug occur on:
- MonoGame 3.8.1

#### What operating system are you using:
- Windows

#### What MonoGame platform are you using:

- iOS

**Comment by Brett208** [comment]
@SimonDarksideJ I think you may just be encountering a name collision. Could you verify adding a using statement to change the name of the Media Player to your project does not fix the issue? (It worked for me on an iOS project I recently ported to MonoGame 3.8.1).

```CSharp
// Required to avoid a collision in the Xamarin iOS SDK 
using XnaMediaPlayer = Microsoft.Xna.Framework.Media.MediaPlayer;

...

XnaMediaPlayer.IsRepeating = true;
XnaMediaPlayer.Play(Sound.Music);
```

Maybe related: https://github.com/MonoGame/MonoGame/issues/3446

**Comment by SimonDarksideJ** [comment]
I'll try that thanks.  Wasn't an issue with 3.8.0, and was only present in 3.8.1

**Comment by corykroll** [comment]
Definitely ran into this and the full `Microsoft.Xna.Framework.Media.MediaPlayer` fixed the issue for me.

**Comment by SimonDarksideJ** [comment]
The issue is that for whatever reason on iOS `MediaPlayer` (even though the iOS library is not referenced and XNA is) is overridden by the iOS compiler.
The solution is to specify it either as a full path resolution `Microsoft.Xna.Framework.Media.MediaPlayer` or apply a named using `using MGMediaPlayer = Microsoft.Xna.Framework.Media.MediaPlayer` to work around the problem.

Not a MonoGame issue, just pesky iOS
